test-requirements.txt

```diff
@@ -14,4 +14,4 @@ pytest-httpbin
 pytest-httpserver
 sanic
 werkzeug==3.0.3
-uvloop==0.19.0
+uvloop==0.21.0b1
```

Comment:
eek -- the test should not be pinned to a different version than the main code, so it's good to change this (tho no pinning would be ideal).

it's been a while since i looked at this code, but IIRC, the tests install everything from setup.cfg as well as everything from the test-requirements.txt.  so given that uvloop is already required by `setup.cfg`, we could delete it from here and avoid any future iterations of the different-pinning problem.  WDYT?

Reply:
I was getting errors due to it not the version of uvloop required for Cython 3. ([This commit)](https://github.com/MagicStack/uvloop/commit/3fba9fab1e8d66c333cf3946d79e29b5bad8fa73). Which is why I pinned it to version `0.21.0b1`. I'll put this PR back in drafts whilst I fix them etc.

Reply:
ahhh, uvloop doesn't have a stable release for cython3 yet.  that explains it.  do we want to wait for upstream to cut a stable release?  that should avoid us needing to do any pinning.  if we don't want to wait, we can just stick a comment in explaining the situation, and Someone™ can remove the comment and the pin in the future....